worker.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var commenter = require('./commenter');
+var redisQueue = require('./redisQueue');
+
+var lastCommentTimestamp = 0;
+
+// Process comment tasks
+// Pause for COMMENT_WAIT milleseconds between comment submissions
+redisQueue.process('comment', function(job, done) {
+    var now = Date.now();
+    var waitRemaining = 0;
+
+    var nextCommentTimestamp = lastCommentTimestamp + parseInt(process.env.COMMENT_WAIT);
+
+    if (nextCommentTimestamp > now) {
+        waitRemaining = nextCommentTimestamp - now;
+    }
+
+    setTimeout(function() {
+        commenter.postPullRequestComment(job.data.commentURL, job.data.comment, job.data.filename, job.data.sha, job.data.line);
+        lastCommentTimestamp = Date.now(); // Use a new, fresh value of Date.now()
```

Comment:
We need to check and see if github rejected our comment.  It looks like right now, if GitHub drops the comment, we aren't saving it.

While we're at it, we should be looking for a `Retry-After` header, and if we get it, we need to wait that long.  Otherwise, we need to temporarily increase our wait time for the next comment.